modules/swagger-codegen/src/test/resources/2_0/petstore.json

```diff
@@ -833,6 +833,10 @@
         "username": {
           "type": "string"
         },
+        "dateOfBirth": {
+          "type": "string",
```

Comment:
Agreed with @cbornet that we should keep Petstore.json as it's.

@jgavris what I would suggest is to add the test case in [petstore-with-fake-endpoints-models-for-testing.yaml](https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing.yaml#L989) instead and then update `swift-petstore.sh` to use `petstore-with-fake-endpoints-models-for-testing.yaml`, which covers a lot more edge cases (you may want to do this in another PR as the edge cases in `petstore-with-fake-endpoints-models-for-testing.yaml` may require some fixes in the Swift generators or templates.